lib/transaction.py

```diff
@@ -425,8 +425,7 @@ def parse_input(vds):
 
 def parse_witness(vds):
     n = vds.read_compact_size()
-    for i in range(n):
-        x = vds.read_bytes(vds.read_compact_size())
+    return list(vds.read_bytes(vds.read_compact_size()).encode('hex') for i in xrange(n))
```

Comment:
Doesn't look like it returned anything before though, also `xrange` doesn't exist in Python 3